management/example_test.go

```diff
@@ -143,9 +152,6 @@ func ExampleConnectionManager_List() {
 			fmt.Printf("\tMulti-Factor Auth Enabled: %t\n", options.MFA["active"])
 		}
 	}
-	// Output: Username-Password-Authentication
```

Comment:
The tests will now run either all using the recordings or without, depending on which make command we run. Because of this we can't expect always the output to be the same so we're removing this to not cause a failure.